lib/utils.js

```diff
@@ -583,20 +589,18 @@ function decorateWithCollation(command, target, options) {
   }
 }
 
-function decorateWithReadConcern(command, coll, options) {
+/**
+ * Applies a read concern to a given command.
+ *
+ * @param {object} command the command on which to apply the read concern
+ * @param {Collection} [target] the parent collection of the operation calling this method
```

Comment:
Should this be `coll` instead of `[target]`?

Reply:
yep! nice catch :smile: